feat: add feature file size limit (#1335)
kubernetes-sigs/node-feature-discovery: docs/usage/customization-guide.md

Label namespace may be specified with `<namespace>/<name>[=<value>]`.

> > **NOTE:** The feature file size limit it 64kB. If the feature file size exceeds
> > the size limit, it won't be parsed.
> 

Reviewer: ```suggestion
> **NOTE:** The feature file size limit it 64kB.  The feature file will be ignored if
> the size limit is exceeded.

```
How about like this?

Author: Sounds better!
